crd/go.mod

```diff
@@ -1,11 +1,18 @@
 module k8s.io/cloud-provider-gcp/crd
 
-go 1.16
+go 1.13
 
 require (
-    k8s.io/api v0.22.2
+	k8s.io/api v0.22.2
 	k8s.io/apimachinery v0.22.2
-    k8s.io/client-go v9.0.0+incompatible // indirect
-	k8s.io/code-generator v0.22.2 // indirect
-    sigs.k8s.io/controller-tools v0.7.0
-)
\ No newline at end of file
+	k8s.io/client-go v0.22.2
+	k8s.io/code-generator v0.22.2
```

Comment:
Change these to be v0.20.0 as well

Reply:
They need to be replaced by the replace statement below since controller-tools will update these fields.  I couldn't find a controller-tools release to generate the exact v0.20.0.  Do you know the controller-tools version being used by the ingress-gce?